src/NuGet/Microsoft.Orleans.Core.nuspec

```diff
@@ -21,7 +21,7 @@
       <dependency id="Newtonsoft.Json" version="9.0.1" />
       <dependency id="System.Collections.Immutable" version="1.3.1" />
       <dependency id="System.Reflection.Metadata" version="1.4.2" />
-      <dependency id="Microsoft.Extensions.DependencyInjection" version="1.1.0" />
+      <dependency id="Microsoft.Extensions.DependencyInjection" version="1.1.1" />
```

Comment:
I think if we're to move from 1.1.0 to 1.1.1 we should do that across the entire code base and in a separate pr.